scripts/bump-version.sh

```diff
@@ -34,10 +34,10 @@ echo "* Bumping version in Dockerfile.rhel"
 sed -i "s/\(version=\)[0-9]*\.[0-9]*\.[0-9]*/\1${NEW_VERSION}/g" Dockerfile.rhel
 check_version Dockerfile.rhel
 
-echo "****************************************************************************************"
-echo "* Don't forget to update homebrew package at https://github.com/kadel/homebrew-odo ! *"
-echo "****************************************************************************************"
+echo "* Bumping version in build/VERSION"
+sed -i "s/\(version=\)[0-9]*\.[0-9]*\.[0-9]*/\1${NEW_VERSION}/g" build/VERSION
+check_version build/VERSION
```

Comment:
Automating this because, apparently, we have forgotten to update the said file since v2.2.1! Just take a look at [this link](https://raw.githubusercontent.com/redhat-developer/odo/main/build/VERSION) before this PR gets merged. On the downside, it might cause issues to users who are prompted to update before a new release is available, but, IMO, let's think of it when someone actually complains about odo asking to update to an unreleased version.

Reply:
Lets not do this. 
Because of our lengthy release process it can more than a week before the binaries are actually available to download. 

Showing users messages to download a new version that might not be available is going to be a horrible experience. 

I would much more prefer the current situation where we sometimes forget to update this, and users don't get notifications about the new version. At least we are not irritating anyone with this.